services/viva-ms/lambdas/submitCompletion.js

```diff
@@ -34,46 +33,59 @@ export async function main(event, context) {
     log.error(
       'Attachment list error',
       context.awsRequestId,
-      'service-viva-ms-submitCompletition-002',
+      'service-viva-ms-submitCompletition-001',
       attachmentListError
     );
-
-    throw attachmentListError;
+    return false;
   }
 
-  log.info(
-    'Answers converted to Attachment List',
-    context.awsRequestId,
-    'service-viva-ms-submitCompletition-003',
-    attachmentList
-  );
-
   const [postCompletionError, postCompletionResponse] = await to(
     vivaAdapter.completion.post({
       personalNumber,
       workflowId: caseItem.details.workflowId,
       attachments: attachmentList,
     })
   );
-
   if (postCompletionError) {
     log.error(
       'post completion error',
       context.awsRequestId,
-      'service-viva-ms-submitCompletition-004',
+      'service-viva-ms-submitCompletition-002',
       postCompletionError
     );
+    return false;
+  }
 
-    throw postCompletionError;
+  if (notCompletionReceived(postCompletionResponse)) {
```

Comment:
Same comments about this function as for `notApplicationReceived `

Reply:
Will be fixed in future task